### DISCUSSION TITLE
Welcome to Fusion-360---Linux-Wine-Version- Discussions!

### DISCUSSION THREAD

**Discussion by cryinkfly** [cryinkfly/Autodesk-Fusion-360-for-Linux#76] [Announcements]
![banner-community](https://user-images.githubusercontent.com/79079633/129786544-ac0e9ea1-b6e5-4ff7-9c69-4671ae71ba6d.png)

## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Atredi** [comment]
@cryinkfly Super excited this exists! I just recently migrated to POP OS and I am looking for a solution to eliminate using a virtual machine. I will test setup tonight. If I want to help you how can I? I am pretty new to Github and Wine but I am sure this effort is worth the support. Thanks for your efforts!

**Reply by cryinkfly** [reply]
@Atredi You can help me by sending me feedback, bug reports, etc.

Please refer: https://github.com/cryinkfly/Autodesk-Fusion-360-for-Linux/blob/main/COMMUNITY.md

**Comment by ManishGru** [comment]
I'm trying to uninstall the fusion360. I don't have any idea how to do that. Could you help me?

**Reply by cryinkfly** [reply]
Yes, you can use my Setup Wizard again for this one or you delete the wineprefix folder itself.

For example (Standard path):

> /$HOME/.wineprefixes/.fusion360

**Reply by ManishGru** [reply]
Thanks!

**Comment by perozu** [comment]
Hi guys,
First off, I was really excited when I found this script and having a working F360 on linux.
My setup is:
MX linux 21 (Debian 11.2)
Wine 7.0 rc5 set as windows 8
In F360, graphics driver on Auto select or DX9

The thing works! :D
Sometimes when doing a sketch, a line doesn't get shows here and there and on my first successful run, the mouse dodn't let go and the object was rotating until i restarted the program.
what are your successful combos for Fusion360 under Linux?

**Reply by cryinkfly** [reply]
> Sometimes when doing a sketch, a line doesn't get shows here and there and on my first successful run, the mouse dodn't let go and the object was rotating until i restarted the program.

@perozu You can also save your project and then you close it an reopen this one. :+1:
